ms/MSOper/MSMetaData.cc

```diff
@@ -3399,6 +3399,8 @@ void MSMetaData::_computeScanAndSubScanProperties(
     uInt chunkSize = nrows/nchunks;
     if (nrows % nchunks > 0) {
         ++chunkSize;
+        // integer division
+        nchunks = nrows/chunkSize + 1;
     }
```

Comment:
ok, that seems to work as far as I can tell.

In that case, the line

nchunks = nrows/chunkSize + 1;

can simply be rewritten as

++nchunks;

do you agree?